### DISCUSSION TITLE
[V2] Multiple select: position selected items below search field

### DISCUSSION THREAD

**Discussion by kokoshneta** [filamentphp/filament#12492] [Help]
### Package

Form builder

### Package Version

v2.17.55

### How can we help you?

When looking at [the V2 docs for multi-select dropdowns](https://filamentphp.com/docs/2.x/forms/fields#multi-select), this is what the component looks like with a few options selected:

![Screenshot of intended multi-select layout](https://user-images.githubusercontent.com/41773797/147613070-cd82703a-fa05-4f29-b0ac-3eb03b542077.png)

When I implement a multi-select dropdown in my Admin Panel, this is what it looks like:

![Screenshot actual multi-select layout](https://github.com/filamentphp/filament/assets/21343099/8cf8ccc4-c58b-4fb5-b1c0-70d681ec3966)

Note how the selected option is positioned _above_ the search field instead of _beneath_ it (and how there’s no separating line between the two).

I would prefer the layout shown in the docs, but I can’t find a method or property to switch. The `Select` component has a [`position()` method](https://github.com/filamentphp/filament/blob/601631d71770d0240ce3b4668f9e69824e12f1a8/packages/forms/src/Components/Select.php#L293), but it’s not documented, and I can’t really figure out what it’s supposed to do. It expects a string or closure parameter, and the string values I’ve tried with have made no discernible difference (including when inspecting the Alpine component on the loaded page).

So how do I get the results after the search field like in the docs?

**Comment by elabx** [comment]
Hi @kokoshneta ! Did you happen to find a solution to this issue? I'm facing the same situation. Thanks!

**Reply by kokoshneta** [reply]
Nope – I think I kind of forgot about it and just lived with it looking a bit wrong…

**Comment by prullmann** [comment]
The `position()` method is forwarded to choices.js, which is used in the background to render the JavaScript version of the Select Field.

[Here](https://github.com/Choices-js/Choices#position) is the documentation, should someone be interested:
> Whether the dropdown should appear above (top) or below (bottom) the input. By default, if there is not enough space within the window the dropdown will appear above the input, otherwise below it.

**Reply by kokoshneta** [reply]
Unfortunately, neither `top` nor `bottom` makes any difference to the actual placement – the method doesn’t seem to have any effect at all.
